shims/0.23/src/main/java/org/apache/hadoop/hive/shims/Hadoop23Shims.java

```diff
@@ -446,16 +478,23 @@ public void shutdown() throws IOException {
     @Override
     public void setupConfiguration(Configuration conf) {
       Configuration config = mr.getConfig();
-      for (Map.Entry<String, String> pair: config) {
+      for (Map.Entry<String, String> pair : config) {
```

Comment:
was this intentional?

Reply:
not, but it's autoformatted by eclipse, actually I liked the change, I always put spaces before and after colon